pages/api/[[...slug]].js

```diff
@@ -42,22 +42,27 @@ const resolvers = {
   Mutation: {
     ...carsMutation,
   },
-
 };
 
 const server = new ApolloServer({ typeDefs, resolvers });
 
 const handler = server.createHandler({
-  path: "/api/graphql",
+  path: '/api/graphql',
   onHealthCheck: () => {
-    console.log("server is running...");
+    console.log('server is running...');
   },
 });
 
 export const config = {
   api: {
     bodyParser: false,
   },
+  cors: {
+    origin: '*',
+    methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7138117)
```suggestion
    methods: "GET,HEAD,PUT,PATCH,POST,DELETE",
```